### DISCUSSION TITLE
借楼讨论，上传下载问题

### DISCUSSION THREAD

**Discussion by bewareilive** [RipplePiam/MobaXterm-Chinese-Simplified#16] [Q&A]
为什么这个经常会左侧文件浏览无法上传或下载，而其他sftp软件就没问题。
还有最新22.1版本，非root用户改root用户777权限文件，修改保存成功了，却弹没权限的报错提示。

**Comment by RipplePiam** [comment]
网上说的是需要一并修改 该文件所在目录的权限。
参考：https://blog.csdn.net/Castlehe/article/details/118675020

**Reply by bewareilive** [reply]
> 网上说的是需要一并修改 该文件所在目录的权限。 参考：https://blog.csdn.net/Castlehe/article/details/118675020

感觉有时是断连接了，重启就可以了。
有时应该是和文章里说法一样，得路径上每一级的文件夹都有权限。

最近遇到上面说的 ，修改保存成功，却弹没权限报错的问题。
后面经过测试，把左边文件浏览器，从sftp改成scp就可以了，并且速度还有提升。
但这样鼠标点击打开文件会有点小卡顿，不太影响
